tests/monotouch-test/OpenGLES/EAGLContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿//
+// EAGLContext Unit Tests
+//
+// Authors:
+//	Vincent Dondain <vidondai@microsoft.com>
+//
+// Copyright 2017 Microsoft
+//
+
+#if !__WATCHOS__
+
+using System;
+#if XAMCORE_2_0
+using Foundation;
+using OpenGLES;
+#else
+using MonoTouch.Foundation;
+using MonoTouch.OpenGLES;
+#endif
+using OpenTK.Graphics.ES20;
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.OpenGLES
+{
+
+	[TestFixture]
+	// we want the test to be available if we use the linker
+	[Preserve (AllMembers = true)]
+	public class EAGLContextTest
+	{
+		[Test]
+		public void PresentRenderBufferTest ()
+		{
+			var obj = new EAGLContext (EAGLRenderingAPI.OpenGLES2);
+
+			Asserts.AreEqual (true, obj.PresentRenderBuffer ((int)RenderbufferTarget.Renderbuffer, 0), "PresentRenderBuffer");
```

Comment:
Would it make sense to add an overload that takes a `RenderbufferTarget` as the target instead of `nuint` (so that you don't have to cast)? Or are there many possible enums?